modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,69 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by Organization Administrators, developers, and end-users; each of these roles has different privileges when connecting applications,
+and there are some restrictions to ensure sensitive data remains secure.
+
+== Organization Administrator
+
+An Organization Administrator is the only role that can view and manage Connected Apps in Access Management.
+
+To manage your connected apps, navigate to Access Management and select the *Applications* tab.
+
+// screenshot of Connected Apps page
+
+The *Connected Apps* page shows a list of applications that you have connected to Anypoint Platform.
+To add an application, select *Add Application*.
+
+=== Create Application
+
+When you create an application as an Organization Administrator, consider the following:
+
+* An organization can own up to 200 connected apps.
+* You can disable or whitelist connected apps.
+* An application is assigned a clientID upon creation; the clientID cannot be modified. However, client secrets can be modified in the *Application Settings* page.
+* When you create an application with the client grant type, the application will continue to work if the application owner's credentials are disabled for any reason. A team or company can assume control of the application's access.
+* An application's actions are logged with an application ID and user ID (for user grant type applications only) in the Audit Log. This empowers administrators to monitor and audit data use.
+
+// screenshot of Create Application interface
+
+When creating an application, you must enter the following information:
+
+* *Application Name*: A descriptive name for your application.
+* *Redirect URIs*: Uniform Resource Identifier that specifies the location of the application. You can specify one per line.
+* *User Grant Types* or *Client Grant Types*: A grant type that aligns with OAuth 2.0 specifications.
+It is important to specify the correct grant type when connecting an application since it will dictate who can access the application. Note that client grant types are only available for first-party applications.
+
+==== User Grants
+
+User grants require authentication from a user, and they are primarily useful for third-party applications. Connected Apps currently supports two types of user grant authentication:
+
+* Authorization: requires the application to produce an authorization code that is accepted by OAuth 2.0.
+If the code should not work indefinitely, you can select the *Refresh Tokens* option, requiring new codes as often as specified in the application.
+* JTW Bearer: grants access using an Application Public Key in a Privacy Enhanced Mail (PEM) that validates the JWT Authorization.
+
+After you select one or more user grant types, you can specify the scope of permissions.
```

Comment:
Scope of permissions is a confusing turn of phrase.
For apps using a user grant type, you specify the list of scopes the app is allowed to request from a user. Each scope represents a set of permissions, identical to how a role represents a set of permissions. Unlike roles, you do not assign the scopes with context, since the context comes from the user who authorizes the app.

For apps using a client grant type, you assign scopes with a context. These assignments can be thought of as identical to assigning an app a role directly. (And they are! just in a OAuth 2.0 flavored / spec compliant way)